### PR TITLE
update the q4 roadmap

### PR DESCRIPTION
**Pull request by diasdavid**
Here is the q4 roadmap stub. Let's go through each of these milestones in the next Monday call and complete them with the tasks that need to be performed.

//cc @jbenet @flyingzumwalt @em-ly

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -100,28 +104,222 @@ UPDATE:
...
+
+### Notes:
+
+NA
+
+##### Expected date of completion: `Week 2 - Oct 24`
+
+
+
+# Milestone - Async Crypto
```
Comment (diasdavid) [inline ROADMAP.md]: @dignifiedquire can you complete this milestone with the tasks that still need to happen for it to be complete?

File: ROADMAP.md [hunk 2]
```diff
@@ -100,28 +104,222 @@ UPDATE:
...
+
+### Notes:
+
+NA
+
+##### Expected date of completion: `Week 3 - Oct 31`
+
+
+
+# Milestone - js-ipfs and js-ipfs-api bundling
```
Comment (diasdavid) [inline ROADMAP.md]: @dignifiedquire can you complete this milestone with the tasks that still need to happen for it to be complete?

File: ROADMAP.md [hunk 3]
```diff
@@ -100,28 +104,222 @@ UPDATE:
...
+
+### Notes:
+
+NA
+
+##### Expected date of completion: `Week 6 - Nov 21`
+
+
+
+# Milestone - Quick Start examples for js-ipfs and js-ipfs-api
```
Comment (diasdavid) [inline ROADMAP.md]: @haadcode  can you complete this milestone with the tasks that still need to happen for it to be complete?

File: ROADMAP.md (hunk 1 above)
Comment (diasdavid) [inline ROADMAP.md]: Exactly, what are the things that need to be done to complete this milestone

File: ROADMAP.md (hunk 2 above)
Comment (diasdavid) [inline ROADMAP.md]: 🛎

File: ROADMAP.md (hunk 3 above)
Comment (diasdavid) [inline ROADMAP.md]: 🛎

File: ROADMAP.md (hunk 1 above)
Comment (dignifiedquire) [inline ROADMAP.md]: They are listed in the PR, should I just copy them into here?

File: ROADMAP.md (hunk 2 above)
Comment (dignifiedquire) [inline ROADMAP.md]: I don't have a concrete list yet, as this depends on going through the current state after async crypto and then see what is still missing.

File: ROADMAP.md (hunk 1 above)
Comment (diasdavid) [inline ROADMAP.md]: Pointing to the PR would be enough

**Comment by diasdavid** [comment]
Going to merge this so that the current available at the root of the repo is up to date.

@dignifiedquire please complete with everything you feel necessary.

**Comment by dignifiedquire** [comment]
@diasdavid sorry for the delay, will add notes as I figure stuff out

**Pull request merged**